command.go

```diff
@@ -37,29 +43,31 @@ func commandParams(cmd *flag.Command) (args []string, msg string) {
 	return
 }
 
-func help(cmd *flag.Command) (err error) {
+func help(cmd *parg.Command) (err error) {
 	if cmd == nil {
-		out.Notification("Usage ::\n\n# Vroomy Package Manager\n" + flag.Help(true))
+		out.Notification("Usage ::\n\n# Vroomy Package Manager\n" + parg.Help(true))
 		return
 	}
 
 	out.Notification("Usage ::\n\n# Vroomy Package Manager\n" + cmd.Help(true))
 	return
 }
 
-func update(cmd *flag.Command) (err error) {
+func update(cmd *parg.Command) (err error) {
 	args, msg := commandParams(cmd)
 	out.Notificationf("Updating %s...", msg)
 
-	if err := v.updatePlugins(args...); err != nil {
+	// Use flag branch instead of config branch
+
+	if err := v.updatePlugins(cmd.StringFrom("branch"), args...); err != nil {
```

Comment:
I'm wondering if this branch override should live in config, so it can be set for individual plugins, all plugins at the config level, or overridden by CLI flag